debezium-core/src/main/java/io/debezium/relational/TableId.java

```diff
@@ -301,20 +310,22 @@ private static String tableId(String catalog, String schema, String table) {
     }
 
     /**
-     * Quotes the given identifier part, e.g. schema or table name.
+     * Quotes the given identifier part, e.g. schema or table name in openingChar and closingChar.
      */
-    private static String quote(String identifierPart, char quotingChar) {
+    private static String quote(String identifierPart, char openingChar, char closingChar) {
         if (identifierPart == null) {
             return null;
         }
 
         if (identifierPart.isEmpty()) {
-            return repeat(quotingChar);
+            return String.valueOf(openingChar) + closingChar;
         }
 
-        if (identifierPart.charAt(0) != quotingChar && identifierPart.charAt(identifierPart.length() - 1) != quotingChar) {
-            identifierPart = identifierPart.replace(quotingChar + "", repeat(quotingChar));
-            identifierPart = quotingChar + identifierPart + quotingChar;
+        if (identifierPart.charAt(0) != openingChar && identifierPart.charAt(identifierPart.length() - 1) != closingChar) {
+            if (openingChar == closingChar) {
+                identifierPart = identifierPart.replace(openingChar + "", repeat(openingChar));
```

Comment:
What's the reason for `+ ""` here?

Reply:
I know it was already in exiting codebase, but IMHO it would be less confusing if you replace it with `String.valueOf(quotingChar)` and code would be more readable.